controllers/chat/moveToAnon.js

```diff
@@ -103,6 +101,9 @@ const moveToAnon = async (req, res) => {
     const createdChannel = await newChannel.create();
     let newStateMemberWithAnonInfo = newChannel.state.members;
 
+    console.log(':::newChannel', newChannel);
+    console.log(':::newStateMemberWithAnonInfo', newStateMemberWithAnonInfo);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Using `console.log` for debugging purposes in production code is not a good practice. It can expose sensitive information and clutter the console output. Consider using a proper logging library that can be configured to log only in development mode or use conditional checks to ensure these logs are only printed during development.
```diff
-    console.log(':::newChannel', newChannel);
-    console.log(':::newStateMemberWithAnonInfo', newStateMemberWithAnonInfo);
+    if (process.env.NODE_ENV === 'development') {
+      console.log(':::newChannel', newChannel);
+      console.log(':::newStateMemberWithAnonInfo', newStateMemberWithAnonInfo);
+    }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->